cmd/virt-launcher/virt-launcher.go

```diff
@@ -477,6 +505,12 @@ func main() {
 			finalShutdownCallback,
 			gracefulShutdownCallback)
 
+		err := runDownwardMetricsVirtioServer(signalStopChan)
+		if err != nil {
+			log.Log.Reason(err).Error("failed to start the DownwardMetrics server")
+			safeCloseChannel(signalStopChan)
```

Comment:
How can the channel be closed multiple times?

Reply:
For instance, if the dmetrics server fails to start and at the same time, the virt-launcher receives a signal to stop